### PR TITLE
feat: promotes settings2 => settings

### PR DESCRIPTION
**Pull request by dzucconi**
OK, so lots to delete.

Notable are the routing updates:

```
  "/settings": "/settings/edit-settings",
  "/profile/edit": "/settings/edit-profile",
  "/user/edit": "/settings/edit-settings",
  "/user/delete": "/settings/delete",
  "/user/saves": "/settings/saves",
  "/user/auctions": "/settings/auctions",
  "/user/shipping": "/settings/shipping",
  "/user/purchases": "/settings/purchases",
  "/user/payments": "/settings/payments",
```

We also are dropping the `/user/refresh` route. Which previously was used to 'refresh' the Backbone user model. Since now all user updates are in Relay, it's no longer needed.

### CODE REVIEW

File: src/desktop/models/user.js [hunk 1]
```diff
@@ -5,107 +5,105 @@
...
-        { display: 'Under $25,000', value: '-1:25000' },
-        { display: 'Under $50,000', value: '-1:50000' },
-        { display: 'Under $100,000', value: '-1:100000' },
-        { display: 'No budget in mind', value: '-1:1000000000000' }
-      ];
-    }
-
-    // This refreshes the user data in the session so that saved data
-    // will stay in sync on reloads
-    refresh(options) {
```
Comment (dzucconi) [inline src/desktop/models/user.js:37]: The only change in this file is deleting this `refresh` method. The rest of the diff is just Prettier updates.

File: src/v2/routes.tsx [hunk 2]
```diff
@@ -78,14 +75,11 @@ export function getAppRoutes(): AppRouteConfig[] {
...
     { routes: partnerRoutes },
     { routes: partnersRoutes },
-    { routes: paymentRoutes },
     { routes: preferencesRoutes },
     { routes: priceDatabaseRoutes },
-    { routes: purchaseRoutes },
     { routes: staticPageRoutes },
     { routes: searchRoutes },
     { routes: settingsRoutes },
-    { routes: shippingRoutes },
```
Comment (dzucconi) [inline src/v2/routes.tsx:88]: All these apps go away now that they are consolidated under `settings`

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Routes added to `routes.tsx` should have a corresponding cypress.js smoke test. See the `cypress/integration` folder for examples.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 7f0ca7e5960744d9948de3a8bcfd4fe7fac54690
</p>

**Review by damassi — DISMISSED** [review]
Lets make sure there's a smoke test for the new app, or file a ticket 👍

**Comment by dzucconi** [comment]
@damassi https://github.com/artsy/force/pull/9296/commits/25aa8f350cf0e4edc9929fea125912319144b8a4

Updated with spec. I'll add some env vars to Circle (just copying over the test user that Integrity uses). And you'll have to grab the env var to run the spec locally.

**Comment by dzucconi** [comment]
It should now; just updated the env there.

**Comment by dzucconi** [comment]
OK, so it _looks_ like CI isn't actually loading the JS for Cypress specs. Rather than deal with that I'm just going to write a more comprehensive suite in Integrity (which will have to be updated for this anyway)

**Pull request merged**